windows_docker_resources/Dockerfile.msvc2019

```diff
@@ -6,11 +6,17 @@
 # $(Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion').ReleaseId
 ARG WINDOWS_RELEASE_ID=1909
```

Comment:
I opted not to remove the WINDOWS_RELEASE_ID arg because I think it is the correct default/fallback but it's only used if WINDOWS_RELEASE_VERSION below is unset.